### PR TITLE
Poppler 23.12.0

### PR DESCRIPTION
**Pull request by mhmdanas**
#### Testing the changes
- I tested the changes in this PR: **YES** (and built all revdeps successfully)

[ci skip]

### CODE REVIEW

File: srcpkgs/poppler/template [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 # Revbump libreoffice on SO version bumps.
```
Comment (sgn) [inline srcpkgs/poppler/template:5]: Remove this line

**Comment by sgn** [comment]
I will update libreoffice, so no revbump for libreoffice.

**Comment by sgn** [comment]
Halo, might you bump calligra-filters

**Comment by mhmdanas** [comment]
@sgn oops, sorry, seems I missed it.

**Pull request merged**